docker-compose.yml

```diff
@@ -104,9 +110,13 @@ services:
       - "discovery.type=single-node"
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
     mem_limit: 2g
+    volumes:
+      - data_elasticsearch:/usr/share/elasticsearch/data
 
   redis:
     image: redis:6.2
+    volumes:
+      - data_redis:/data
```

Comment:
See other comment, probably not needed / makes our life easier if we avoid this (we only need redis as a celery result backend)

Reply:
Adding --volumes to the down command handles this.